### PR TITLE
Ease of use updates

### PR DESCRIPTION
**Pull request by sdzhong**
1. Added logging
2. Consistent use of env vars
3. User input for team mapping option
4. Matching role of source onprem user for newly created cloud users

### CODE REVIEW

**Comment by sdzhong** [comment]
[sentry_member_team_map_withoutSCIM.py.log](https://github.com/sentry-demos/rest-api-scripts/files/8134546/sentry_member_team_map_withoutSCIM.py.log)
Example log `sentry_member_team_map_withoutSCIM.py.log` output attached.

**Comment by sdzhong** [comment]
Got the OK from @adam000034 via Slack to Merge this PR.

File: sentry_member_team_map_withoutSCIM.py [hunk 1]
```diff
@@ -72,77 +72,96 @@ def update_team_reg(self, memberid, teamname):
...
-                    updated_ids_dict[member.get('email')] = cloudmember.get('id')
-                    found = 1
-                    break
-
-        #Make a new user if not found in new org, email must match what is on Okta
+            if (member.get('email') == (cloudmember.get('email'))):
+                member['id'] = cloudmember.get('id')
+                #update id in common dictionary to use pre-existing cloud member's id
+                updated_ids_dict[member.get('email')] = cloudmember.get('id')
+                found = 1
```
Comment (ndmanvar) [inline sentry_member_team_map_withoutSCIM.py:119]: @sdzhong in the future, use booleans :)

File: sentry_member_team_map_withoutSCIM.py [hunk 2]
```diff
@@ -72,77 +72,96 @@ def update_team_reg(self, memberid, teamname):
...
+                if team in cloud_teams:
+                    sentry_cloud.update_team_reg(member['id'], team)
+                    logger.info("Cloud user %s has been assigned to team %s" % (member.get("email"), team))
+                else:
+                    logger.info("Team %s does not exist in cloud! Could not add user %s" % (team, member.get("email")))
+    else:
+        logger.info("Skipped assigning cloud users to corresponding onprem teams")
+
+    logger.info("<<< Script completed")
+    print("\nScript completed. Log available in ./%s" % (os.path.basename(__file__)+'.log'))
```
Comment (ndmanvar) [inline sentry_member_team_map_withoutSCIM.py:167]: @sdzhong - very minor, insert newline at end in the future

**Pull request merged**